crowdsourcing/viewsets/requester.py

```diff
@@ -1,12 +1,22 @@
 __author__ = 'dmorina, asmita, megha'
-
 from crowdsourcing.serializers.requester import *
+from crowdsourcing.serializers.project import *
 from rest_framework import viewsets
-
+from rest_framework.decorators import detail_route
+from rest_framework.response import Response
+from crowdsourcing.models import Module
 class RequesterViewSet(viewsets.ModelViewSet):
     from crowdsourcing.models import Requester
     queryset = Requester.objects.all()
     serializer_class = RequesterSerializer
+    @detail_route(methods = ['GET'])
+    def portfolio(self, request, *args, **kwargs):
+        requester = self.get_object()
+        modules = requester.module_set.all().filter(deleted=False,status=4)
+        serializer = ModuleSerializer(modules,many = True,fields = ('id', 'name', 'icon', 'project', 'categories',
```

Comment:
ModuleSerializer(modules,many = True,fields = ....
here your first parameter is not named but the rest are